board/miyoo/boot/firstboot

```diff
@@ -756,10 +756,20 @@ if $CHANGE_FS_P5; then
 	elif test "${P5_FS_TYPE}" == "EXT4"; then
 		sed -i "s|\(::sysinit:/bin/mount /dev/mmcblk0p5\).*|\1 "${ROMS}" -o rw,noatime|" /etc/inittab
 	fi
-	mount -o remount,ro / /
-	cat /etc/inittab >> ${LOG}
+
 fi
 
+#Check if btrfs module needed
+if test "${P4_FS_TYPE}" != "BTRFS" && test "${P5_FS_TYPE}" != "BTRFS" ; then
+  mount -o remount,rw / /
+  sed -i 's|^::sysinit:/sbin/modprobe btrfs|#&|' /etc/inittab
+  mount -o remount,ro / /
+  cat /etc/inittab >> ${LOG}
+  modprobe -r btrfs 2>&1 | tee -a "${LOGS}"
+fi
+
```

Comment:
- the FIRSTBOOT can be rerun from options.cfg (see https://github.com/MiyooCFW/buildroot/pull/175), so you have to uncomment it if commented otherwise and BTRFS have been picked again. 

- rm extra newlines